src/js/components/pages/About/About.js

```diff
@@ -32,7 +33,7 @@ export default class About extends Component {
 
           <Navbar light />
 
-          <Content></Content>
+          <Content />
```

Comment:
Please get back AboutMe component and remove all changes in Content component.
Content is only a wrapper of content. So in this line should be
```<Content>
    <AboutMe />
</Content>```